features/step_definitions/admin_policy_tagging_steps.rb

```diff
@@ -0,0 +1,8 @@
+Then(/^I can tag it to some policies$/) do
+  select policy_1["title"], from: 'Policies'
+  click_button 'Save'
+  publication = Publication.last
+
+  assert_path admin_publication_path(publication)
+  assert_equal [policy_1["content_id"]], publication.policy_content_ids
+end
```

Comment:
I've seen this pattern followed in some other apps, but not so much in whitehall to now. Can you help me understand the advantage of wrapping the capybara actions into helpers in this way? I can understand if it's to reduce duplication, wrapping common actions into a re-usable method (e.g. `begin_drafting_document`). But that doesn't feel necessary here as the step definition itself already entirely encapsulates the action of tagging the document to a policy and that is not performed anywhere else. It makes me uneasy because it adds another level of indirection to cucumber features which already have enough indirection (feature file -> cucumber steps -> helpers -> capybara actions).